ledger-core/v2/virtual/object/object.go

```diff
@@ -91,16 +112,57 @@ func (sm *SMObject) Init(ctx smachine.InitializationContext) smachine.StateUpdat
 	sm.ImmutableExecute = smsync.NewSemaphore(30, "immutable calls").SyncLink()
 	sm.MutableExecute = smsync.NewSemaphore(1, "mutable calls").SyncLink() // TODO here we need an ORDERED queue
 
+	sm.alreadySendVStateRequest = false
```

Comment:
default bool value is false already

Reply:
maybe stateRequestSent would be better name

Reply:
fixed